doc/process/special_access_to_build_resources.md

```diff
@@ -0,0 +1,77 @@
+# Introduction
+
+There are a number of cases were we would like to provide access to
+community machines and or jenkins jobs to people who are not part
+of the build working group.  Examples include:
+
+* test working group team members so that they can add/debug tests to/on 
+  community test infrastructure.
+* benchmark working group members so they can add benchmark jobs and
+  or experiement with benchmarking infrastructure in advance of the
+  final configuration being added to ansible.
+* PR's authors so they can debug failures across platforms in cases
```

Comment:
I think we want to leave the door open to non-collaborators as some of the options do allow for that.